apps/src/aichat/views/aichatView.module.scss

```diff
@@ -69,4 +70,5 @@ $column-min-width: 200px;
 
 .instructions {
   @extend %panel-section;
+  height: 90%;
```

Comment:
Can we set a fixed margin instead of % so it stays consistent across screen sizes? I'd be curious how this looks on very small and very large screens

Also nit: can we include a comment for why we're setting this height?